editor/src/components/canvas/controls/select-mode/scene-label.tsx

```diff
@@ -92,6 +93,12 @@ const SceneLabel = React.memo<SceneLabelProps>((props) => {
   const offsetX = scaledFontSize
   const borderRadius = 3 / scale
 
+  const mode = useEditorState(
```

Comment:
general tip that since this value is _only_ used in a callback, if you make this a useRefEditorState, then you can use isSelectModeWithArea(mode.current) in the callback, and then you don't need to incur a re-render of scene label on mode change, and you don't need to blow the reference equality of onMouseMove either!